.gitignore

```
@@ -8,3 +8,5 @@ vars/*.yml
 secrets/prod
 secrets/stg
 secrets/dev
+
+secrets/secrets.yaml
```

What do you think about gitignoring whole secrets/?
```suggestion
secrets/*
```

---

But we have there the templates and README.md

---

You're right. Go ahead & merge.